recipe/meta.yaml

```diff
@@ -64,9 +76,21 @@ about:
   license_family: MIT
   license_file: LICENSE
   summary: Python dependency management and packaging made easy
+  description: |
+    Poetry is a tool for dependency management and packaging in Python. 
+    It allows you to declare the libraries your project depends on and it 
+    will manage (install/update) them for you. Poetry offers a lockfile 
+    to ensure repeatable installs, and can build your project for distribution.
   doc_url: https://python-poetry.org/docs/
   dev_url: https://github.com/python-poetry/poetry/
 
 extra:
   recipe-maintainers:
-    - ocefpaf
+    - Arun Babu Neelicattu
+    - Bjorn Neergaard
+    - Branch Vincent
+    - Bryce Drennan
+    - Daniel Eades
+    - Randy Döring
+    - Steph Samson
+    - finswimmer
```

Comment:
Instead of names there should be nicknames https://github.com/conda-forge/poetry-feedstock/blob/a6f70717bd2734177b11b7cfd4009ee716fa5780/recipe/meta.yaml#L95-L99

Reply:
I've updated this